core/idotmatrix/common.py

```diff
@@ -101,7 +123,7 @@ def setJoint(self, mode):
         Returns:
             _type_: byte array of the command which needs to be sent to the device
         """
-        try:
+        try:    
```

Comment:
Could you please clean this up? Seems like you have pressed the TAB key.

Reply:
Done